### ISSUE TITLE
Banner image is not taken into account

### ISSUE THREAD

**Issue by Popolechien** [openzim/youtube#342]
Looking at several of our latest Youtube channels I see that when the recipe selects for playlists rather than the entire channel their banner image is never included in the corresponding zim file.

See, e.g.:
| Youtube  | Zim |
| ------------- | ------------- |
|[S2 Underground YT](https://www.youtube.com/@s2underground)|[S2 zim](https://library.kiwix.org/viewer#s2underground_en_all_2024-09/index.html)|
|[Voice of America YT](https://www.youtube.com/@voalearningenglish)|[VoA zim](https://library.kiwix.org/#lang=fra&q=apprenons)|
|[Canadian prepper YT](https://library.kiwix.org/viewer#canadian_prepper_winterprepping_en_2024-09/index.html)|[Canadian prepper ZIM](https://www.youtube.com/@CanadianPrepper/playlists)|

**Comment by benoit74** [comment]
Thank you for reporting, this is indeed a problem.

**Comment by benoit74** [comment]
This is in fact "normal" behavior, see https://github.com/openzim/youtube/blob/0b9bed587f5a2cb890d272aee848b89cb2fb5fe8/CHANGELOG#L137-L139

We can have a look again whether google changed again the API and we can retrieve the banner again.

**Comment by kelson42** [comment]
> Is this really worth it? I'm not against this, but it does look a bit fragile. And setting the banner image manually is not "that hard", so probably not the most priority issue to solve. That being said, I'm not against someone proposing a PR on this issue, just wondering which focus we should put on this issue.

Yes, fragile... but seems to be the best we can do... and the banner is pretty important!

**Comment by benoit74** [comment]
This will come back with move to full yt-dlp in fact (see #177), yt-dlp nicely provides the banner.
